Resolve problem on First navigation with Recurrence settings on. (#3435)
the-events-calendar/the-events-calendar: readme.txt

= [5.3.3] TBD =

* Fix - Navigation for the Views will no longer use current browser URL as previous url, preventing problems on shortcodes.

> * Tweak - Include a filter `tribe_events_views_v2_view_page_reset_ignored_params` for prevent certain params to interfere with pagination resetting. [FBAR-222]

Reviewer: ```suggestion
* Tweak - Include the `tribe_events_views_v2_view_page_reset_ignored_params` filter to prevent certain params from interfering with pagination resetting. [FBAR-222]
```
